README.md

```diff
@@ -1,5 +1,10 @@
 # Canonical Store Api - Python package
 
+![](https://github.com/canonical-web-and-design/canonicalwebteam.store-api/workflows/Test%20live%20APIs/badge.svg)
+![](https://github.com/canonical-web-and-design/canonicalwebteam.store-api/workflows/Tests/badge.svg)
+![](https://github.com/canonical-web-and-design/canonicalwebteam.store-api/workflows/Publish/badge.svg)
```

Comment:
I think it would be nice to both give these badges titles and also make them into links, just like the "Code coverage" badge below.

Reply:
I think it will show the workflow name by default but I agree about the link. I did this changes.